app.go

```diff
@@ -194,10 +198,8 @@ func executeAdd(cmd command) bool {
 
 	// Add the amount
 	log.Infof("Adding %.2f to %s", amount, cmd.UserID)
-	if err := accountStore.Accounts[cmd.UserID].AddFunds(amount); err != nil {
-		log.Error(err.Error())
-		return false
-	}
+	accountStore.Accounts[cmd.UserID].AddFunds(amount)
+
 	balance := accountStore.Accounts[cmd.UserID].Balance
 	log.Infof("New balance for %s is %.2f", cmd.UserID, balance)
```

Comment:
![screenshot from 2017-01-20 02-54-31](https://cloud.githubusercontent.com/assets/5499880/22140661/abef452e-dea2-11e6-97dc-4126665cf6e4.png)

Probably leftover code from when balance was a float64. Got it.